City-of-Helsinki/drupal-helfi-platform-config
# UHF-3364: hide react & share until statistics cookies accepted

How to test:

- Set up any site
- Checkout this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-3364_react_and_share_cookie_compliance`
- Checkout correct branch of the HDBT theme: `composer require drupal/hdbt:dev-UHF-3364_react_and_share_cookie_compliance` (see PR: https://github.com/City-of-Helsinki/drupal-hdbt/pull/218)
- Clear cache: `make drush-cr`
- Set up a page with React & Share component
  - Navigate on the page using incognito mode
  - See the cookie notification where the React & Share component should be
  - Accept cookies and see the notification get replaced by the React & Share component
